src/reactA11yNoOnchangeRule.ts

```diff
@@ -0,0 +1,64 @@
+import * as ts from 'typescript';
+import * as Lint from 'tslint';
+
+import {ErrorTolerantWalker} from './utils/ErrorTolerantWalker';
+import {ExtendedMetadata} from './utils/ExtendedMetadata';
+import {getJsxAttributesFromJsxElement} from './utils/JsxAttribute';
+
+/**
+ * Implementation of the react-a11y-no-onchange rule.
+ */
+export class Rule extends Lint.Rules.AbstractRule {
+
+    public static metadata: ExtendedMetadata = {
+        ruleName: 'react-a11y-no-onchange',
+        type: 'functionality',
+        description: 'For accessibility of select menus, onChange event handler should not be used with the select element.',
```

Comment:
For users who are new to accessibility concerns, it's confusing why the `onChange` event handler isn't allowed with the select element. Could you please add a `rationale` mentioning the preference to use `onBlur`?